mypy/config_parser.py

```diff
@@ -28,20 +28,20 @@ def parse_version(v: str) -> Tuple[int, int]:
     m = re.match(r'\A(\d)\.(\d+)\Z', v)
     if not m:
         raise argparse.ArgumentTypeError(
-            "Invalid python version '{}' (expected format: 'x.y')".format(v))
+            f"Invalid python version '{v}' (expected format: 'x.y')")
     major, minor = int(m.group(1)), int(m.group(2))
     if major == 2:
         if minor != 7:
             raise argparse.ArgumentTypeError(
-                "Python 2.{} is not supported (must be 2.7)".format(minor))
+                f"Python 2.{minor} is not supported (must be 2.7)")
     elif major == 3:
         if minor < defaults.PYTHON3_VERSION_MIN[1]:
             raise argparse.ArgumentTypeError(
-                "Python 3.{0} is not supported (must be {1}.{2} or higher)".format(minor,
+                "Python 3.{} is not supported (must be {}.{} or higher)".format(minor,
```

Comment:
I think it might be cleaner to unpack `PYTHON3_VERSION_MIN` in the previous line and then use an f-string here